docs2/site/docs/migrations/migration5.md

```diff
@@ -168,9 +187,9 @@ automatically-generated graph types:
 |------------------|----------------------------------------------------------------------|------------------|
 | (constructor)    | Configures graph properties and adds fields                          | Configuring graph after default initialization is complete |
 | ConfigureGraph   | Configures default graph properties prior to applying attributes     | Applying a different default naming convention, such as appending "Input" or "Model" |
-| GetRegisteredProperties | Returns the set of properties to be automatically configured  | Filtering internal properties; sorting the property list |
+| GetRegisteredMembers | Returns the set of properties, methods and fields to be automatically configured | Filtering internal properties; sorting the property list; including fields; excluding methods |
 | ProvideFields    | Returns a set of generated fields                                    | Adding additional fields to the generated set |
-| CreateField      | Creates a `FieldType` from a `PropertyInfo`                          | Applying custom behavior to field generation |
+| CreateField      | Creates a `FieldType` from a `MemberInfo`                            | Applying custom behavior to field generation |
 
 If you utilize dependency injection within your schema, you can register your custom graph
```

Comment:
```suggestion
Note that if you override `GetRegisteredMembers` to include private methods or fields, you
may also need to override `ParseDictionary` as well.

If you utilize dependency injection within your schema, you can register your custom graph
```

Reply:
I'll add this in now but probably it will change later once I redo ParseDictionary.